services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitFormNew.tsx

```diff
@@ -55,15 +56,14 @@ export enum EsupFormMode {
 }
 
 interface ExplosivesPermitFormProps {
-  closeModal: () => void;
+  onSubmit: any;
```

Comment:
I considered renaming this file to just `ExplosivesPermitForm` and taking off the `New` but I thought about how much more confusing the diff would be. Another day!